### ISSUE TITLE
Building navigation with multiple projects

### ISSUE THREAD

**Issue by digitalcosmos** [Geaz/sharpDox#33]
I'm trying to build documentation from a '.sdnav' which references 2 projects.
I'm getting the following crash:
```
2015-05-01 12:40:13,120: ERROR – System.InvalidOperationException: Sequence contains more than one element
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   at SharpDox.Plugins.Html.Steps.CreateHtmlStep.GetSearchIndex()
   at SharpDox.Plugins.Html.Steps.CreateHtmlStep.RunStep()
   at SharpDox.Plugins.Html.HtmlExporter.Export(SDProject sdProject, String outputPath)
   at SharpDox.Build.Context.Step.ExportStep.RunAllExporters(SDProject sdProject)
   at SharpDox.Build.Context.Step.ExportStep.RunStep(SDProject sdProject)
   at SharpDox.Build.Context.BuildContext.StartBuild()
```

Upon further investigation the cause is at line 55 in `CreateHtmlStep.cs`
```
  var searchIndexTemplate = new SearchIndexTemplate() { Articles = StepInput.SDProject.Articles.GetElementOrDefault(StepInput.CurrentLanguage), Repository = StepInput.SDProject.Repositories.SingleOrDefault().Value };
```
`SingleOrDefault` method throws exception because `Repositories` contains 2 entries.

I'm sorry that I cannot fix it myself because I'm not familiar with the project (is it supposed to have only one repository or should that be 'FirstOrDefault()'?). 

Thanks for the help and great work on the project!!!

**Comment by Geaz** [comment]
This is in fact a bug. Changing to 'FirstOrDefault' would work, but the search wouldn't work as expected anymore. Will fix this in the next release!

**Comment by Geaz** [comment]
Fixed in v1.0rc
